docs/content/configuration/index.md

```diff
@@ -162,7 +162,7 @@ The Druid servers emit various metrics and alerts via something we call an Emitt
 
 #### Graphite Emitter
 
-To use graphite as emitter set `druid.emitter=graphite`. For configuration details please follow this [link](https://github.com/druid-io/druid/tree/master/extensions/graphite-emitter/README.md).
+To use graphite as emitter set `druid.emitter=graphite`. For configuration details please follow this [link](https://github.com/druid-io/druid/tree/master/extensions/graphite-emitter/README.html).
```

Comment:
please merge https://github.com/druid-io/druid/pull/2698 first. The correct link is under extensions-contrib/graphite.html